modules/operator/scripts/configure_agent.template.sh

```diff
@@ -3,7 +3,7 @@
 # Copyright 2020, Oracle Corporation and/or affiliates.  
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
-echo 'configuring agent'
+echo 'Configuring agent'
```

Comment:
Can we change this to `Configuring Platform Agent`?

Reply:
On this and other points - can do, initial changes were just to capitalise before changing grammar/wording.